dangerzone/gui/main_window.py

```diff
@@ -119,6 +119,12 @@
 </p>
 """
 
+DOWNLOAD_NEW_CONTAINER_IMAGE_MSG = """\
+<p>A new version of the secure sandbox is available and should be downloaded.
+ Should we proceed now?</p>
+<p>Keeping the sandbox up to date helps avoiding security vulnerabilities.
+ It's recommended to always apply updates.</p>
```

Comment:
I want to address two things here:

1. Mention that the size of the download is usually around 500MiB, so that the users can make a more educated guess.
2. Mention that if they skip the update, they will be asked the next time they use Dangerzone.

Proposed text:

```suggestion
<p>A new version of the secure sandbox is available and should be downloaded.
The download size is usually around 500MiB.
Should we download and install it now, or skip it and ask the next time you run Dangerzone?</p>
<p>Keeping the sandbox up to date helps avoiding security vulnerabilities.
 It's recommended to always apply updates.</p>
```